src/ontology/uberon-edit.obo

```diff
@@ -190087,6 +190087,24 @@ is_a: UBERON:0001145 {source="FMA"} ! ovarian vein
 intersection_of: UBERON:0001145 ! ovarian vein
 intersection_of: drains UBERON:0002119 ! left ovary
 
+[Term]
+id: UBERON:0700019
+name: parallel fiber
+def: "A parallel fiber results from the axon of a cerebellar granule cell ascending through the molecular layer where it synapses on the dendritic branches of Purkinje cells and stellate cells." [GO:1990032] {comment="doi:10.1007/978-94-007-1333-8_63"}
+comment: This a grouping class for parallel fibers in any species. If annotating a parallel fiber in any species other than teleost fish, consider using 'parallel fiber, bifurcated'.
+is_a: UBERON:0006134 ! nerve fiber
+relationship: has_part CL:0001031 ! cerebellar granule cell
```

Comment:
I am not sure if I agree with this axiom. The definition of has_part is not very clear either: a core relation that holds between a whole and its part.
The way I understand is that the whole (but not all!) cerebellar granule cell are part of the fiber. However, in this case, only the axioms are part of the fiber. Overlaps is more general, but that's the object property I would consider using in this case.
However, as @dosumis [requested this axiom](https://github.com/obophenotype/cell-ontology/issues/2012#issue-1747510248), I will defer the review of it to him.

Reply:
> I am not sure if I agree with this axiom. The definition of has_part is not very clear either: a core relation that holds between a whole and its part.
> The way I understand is that the whole (but not all!) cerebellar granule cell are part of the fiber. However, in this case, only the axioms are part of the fiber. Overlaps is more general, but that's the object property I would consider using in this case.
> However, as @dosumis [requested this axiom](https://github.com/obophenotype/cell-ontology/issues/2012#issue-1747510248), I will defer the review of it to him.

**overlaps** seems to be a reasonable relation in this instance. 
@dosumis, kindly advise if there is a compelling reason to use 'has part' instead of overlaps here.

Reply:
Checking a similar relationship:
 '[dentate gyrus mossy fiber](http://purl.obolibrary.org/obo/GO_0044302)' has
`SubClass Of 'part of' some 'granule cell'`.

Given that, here is the updated proposal:

'parallel fiber' `SubClass Of 'part of' some 'cerebellar granule cell'`

Reply:
While the majority of parallel fibers are unmyelinated, there are some myelinated parallel fibers [PMID:15869526](https://pubmed.ncbi.nlm.nih.gov/15869526). As the myelin sheath is not part of the 'cerebellar granule cell', I would not use 'part of'.